provision/acc_provision/templates/calico-installer-cr.yaml

```diff
@@ -5,6 +5,10 @@ metadata:
   name: installer-calico-cni
 spec:
   config:
+    connectivityCheck:
```

Comment:
This entire "connectivityCheck" section should only exist if config.connectivity_checker field exists in the input file. Since this is optional to set by the user. No "else" required

Reply:
addressed